packages/renderers/renderer-lit/server.js

```diff
@@ -2,6 +2,14 @@ import './server-shim.js';
 import '@lit-labs/ssr/lib/render-lit-html.js';
 import { LitElementRenderer } from '@lit-labs/ssr/lib/lit-element-renderer.js';
 
+const reservedReactProperties = new Set([
```

Comment:
`reservedAstroProperties`?

Reply:
Is this defined anywhere else in the Astro codebase? It'd be good to import it so it never gets out of sync.

Reply:
Nope, this is the only WC renderer we have at the moment.

Reply:
What I mean is if these properties are special to Astro, there must be a place in the Astro code that uses them. Are they just spread out in individual checks in the property handling code?

Reply:
Not sure on the JSX side like className or anything, but should we include the hydrators e.g. `client:idle`?

Reply:
These properties are not related to Astro at all. `client:idle`, etc don't make it into the renderers, I believe. So shouldn't have to worry about those. This looks good to me.

Reply:
Then should this just be removed?

Reply:
Yes, I think this should be removed. We don't have special handling for any of these. The few I can think (`client:*`, `class:list`) are never passed to renderers.